include/database.h

```diff
@@ -25,12 +25,19 @@
 #include <memory>
 #include <map>
 #include <stdexcept>
+#include <functional>
 
-class null_argument : std::runtime_error {
+#include "../src/fty_alert_list_classes.h"
```

Comment:
That shouldn't be needed!

Reply:
Topdir/src should be in includedir, otherwise zproject must be fixed. But for master